distribution/tools/server-cli/src/main/java/org/elasticsearch/server/cli/JvmOptionsParser.java

```diff
@@ -72,50 +75,73 @@ public static void main(final String[] args) throws InterruptedException, IOExce
                 "Expected two arguments specifying path to ES_PATH_CONF and plugins directory, but was " + Arrays.toString(args)
             );
         }
+        try {
+            Path configDir = Paths.get(args[0]);
+            Path pluginsDir = Paths.get(args[1]);
+            Path tmpDir = Paths.get(System.getenv("ES_TMPDIR"));
+            String envOptions = System.getenv("ES_JAVA_OPTS");
+            var jvmOptions = determineJvmOptions(configDir, pluginsDir, tmpDir, envOptions);
+
+            Launchers.outPrintln(String.join(" ", jvmOptions));
+        } catch (UserException e) {
+            Launchers.errPrintln(e.getMessage());
+            Launchers.exit(e.exitCode);
+        }
+    }
+
+    /**
+     * Determines the jvm options that should be passed to the Elasticsearch Java process.
+     *
+     * <p> This method works by joining the options found in {@link SystemJvmOptions}, the {@code jvm.options} file,
+     * files in the {@code jvm.options.d} directory, and the options given by the {@code ES_JAVA_OPTS} environment
+     * variable. After the options are read, they are passed to a java subprocess which prints out the final options.
```

Comment:
The last sentence throws me off, since we pass to the subprocess in the `main()` method, not here.

Reply:
The subprocess doesn't happen in main, it happens down inside `JvmErgonomics`, which internally calls `JvmOption.findFinalOptions`. I'll just remove this line though as I see how it is confusing.